gluoncv/model_zoo/model_store.py

```diff
@@ -13,15 +13,15 @@
     ('daf8181b615b480236fcb8474545077891276945', 'ssd_512_vgg16_atrous_voc'),
     ('9c8b225a552614e4284a0f647331bfdc6940eb4a', 'ssd_512_resnet50_v1_voc'),
     ('2cc0f93edf1467f428018cc7261d3246dfa15259', 'ssd_512_resnet101_v2_voc'),
-    ('ce85f11ce208d8f5adc9da52753b717db73a2c6e', 'cifar_resnet20_v1'),
-    ('f920e8b8186132e64518a4502c57e381ef336b72', 'cifar_resnet20_v2'),
-    ('aa01e54160e2c92b8dee2319f1c2a8ea0e327ed1', 'cifar_resnet56_v1'),
-    ('81edd76526cc736df96b165eb6f8850bcad9d6d6', 'cifar_resnet56_v2'),
-    ('3a9d8dda8e54f06735b1d61b85002d00c8cd5005', 'cifar_resnet110_v1'),
-    ('d7bacf7cb1b5e19348941e6d486b9262efe78b15', 'cifar_resnet110_v2'),
-    ('ebc52a3cb68fb9a14f59944b510cc682675a70cc', 'cifar_wideresnet16_10'),
-    ('0ef2c7bec9c2c48b8ff52440ae84869ce2db20b1', 'cifar_wideresnet28_10'),
-    ('5369ff00708900aa34c85088ac2c15fd42f0b243', 'cifar_wideresnet40_8'),
+    ('121e1579d811b091940b3b1fa033e1f0d1dca40f', 'cifar_resnet20_v1'),
+    ('4f2d18804c94f2d283b8b45256d048bd3d6dd479', 'cifar_resnet20_v2'),
+    ('2fb251e60babdceb103e9659b3baa0dea20a14d7', 'cifar_resnet56_v1'),
+    ('0a3e74104ec7bcfffefe2d9d5cc1f8e74311ec51', 'cifar_resnet56_v2'),
+    ('a0e1f860475bf5369f6da07e0c2e03a4ae9cff9c', 'cifar_resnet110_v1'),
+    ('bf160f8b3cb3884a1ea871739f3c8e151e114159', 'cifar_resnet110_v2'),
+    ('7c07b5ba6e850f9c37ca1e57c0a2e529455cc2e4', 'cifar_wideresnet16_10'),
+    ('4a3466aadd4c3ddbcb968bca862d0e59d6f15ec1', 'cifar_wideresnet28_10'),
+    ('085ca2afabbe0ddfe87d0edc5408bcfcfbffd414', 'cifar_wideresnet40_8'),
```

Comment:
Is the performance different than reported in the model zoo docs?
http://gluon-cv.mxnet.io/model_zoo/index.html

Reply:
They all achieve better results. The doc's also updated, please check the pre-built temporary page.

Reply:
I see it now :)